gef-ui/templates/partials/ukef-cover-start-date.njk

```diff
@@ -0,0 +1,154 @@
+            {{ govukBackLink({
+                text: "Back",
+                href: "/gef/application-details/" + applicationId + "/confirm-cover-start-date",
+                attributes: {
+                'data-cy': 'back-link'
+                }
+            }) }}
+        </div>
+    </div>
+
+    {% if isUkefReviewPositive %}
+        <div class="govuk-grid-row govuk-body">
+            <div class="govuk-grid-column-two-thirds">
+                <form method="POST" autocomplete="off">
+
+                    {# Custom cover start date #}
+                    {% set coverStartDate %}
+                        {{ govukDateInput({
+                            id: "ukefCoverStartDateInput",
+                            name: "ukefCoverStartDateInput",
+                            fieldset: {
+                                legend: {
+                                    text: "Enter a cover start date",
+                                    classes: "govuk-fieldset__legend--m"
+                                }
+                            },
+                            hint: {
+                                html: "<p class='govuk-!-margin-0'>For example, 12 03 2021. This must be within 3 months after you submit the inclusion notice.</p>"
+                            },
+                            errorMessage: errors and errors.fieldErrors.ukefCoverStartDateInput and {
+                            text: errors.fieldErrors.ukefCoverStartDateInput.text,
+                                attributes: {
+                                "data-cy": "ukefCoverStartDateInput-error"
+                                }
+                            },
+                            items: [
+                                {
+                                    Label: "Day",
+                                    classes: errors and errors.fieldErrors['coverStartDate-day'] and "govuk-input--error govuk-input--width-2" or "govuk-input--width-2",
+                                    name: "day",
+                                    value: facility.date,
+                                    attributes: {
+                                        "data-cy": "ukef-cover-start-date-day"
+                                    }
+                                },
+                                {
+                                    Label: "Month",
+                                    classes: errors and errors.fieldErrors['coverStartDate-month'] and "govuk-input--error govuk-input--width-2" or "govuk-input--width-2",
+                                    name: "month",
+                                    value: facility.month,
+                                    attributes: {
+                                        "data-cy": "ukef-cover-start-date-month"
+                                    }
+                                },
+                                {
+                                    Label: "Year",
```

Comment:
not sure if it's causing issues but label should be lowercase 👍

Reply:
Done